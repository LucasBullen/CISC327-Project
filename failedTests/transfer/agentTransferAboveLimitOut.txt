1,7c1,7
< Select mode: atm or agent.
< Logged into agent mode.
< Enter first account number.
< Enter second account number.
< Enter amount to transfer.
< Incorrect amount format. Transfer cancelled.
< Logged out.
---
> Select mode: atm or agent.
> Logged into agent mode.
> Enter number of account to deposit into.
> Enter number of account to transfer to.
> Enter amount (in cents) to transfer.
> Transfer above $999,999.99 limit. Transfer cancelled.
> Logged out.
