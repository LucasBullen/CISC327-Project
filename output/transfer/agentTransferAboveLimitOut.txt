Select mode: atm or agent.
Logged into agent mode.
Enter first account number.
Enter second account number.
Enter amount to transfer.
Incorrect amount format. Transfer cancelled.
Logged out.
