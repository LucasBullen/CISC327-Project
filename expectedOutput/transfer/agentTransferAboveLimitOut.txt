Select mode: atm or agent.
Logged into agent mode.
Enter number of account to deposit into.
Enter number of account to transfer to.
Enter amount (in cents) to transfer.
Transfer above $999,999.99 limit. Transfer cancelled.
Logged out.
